structure(list(table_name = c("alt_allele", "analysis", "analysis_description", 
"assembly", "assembly_exception", "attrib_type", "coord_system", 
"density_feature", "density_type", "dependent_xref", "ditag", 
"ditag_feature", "dna", "dna_align_feature", "dnac", "exon", 
"exon_stable_id", "exon_transcript", "external_db", "external_synonym", 
"gene", "gene_archive", "gene_attrib", "gene_stable_id", "go_xref", 
"identity_xref", "interpro", "karyotype", "map", "mapping_session", 
"mapping_set", "marker", "marker_feature", "marker_map_location", 
"marker_synonym", "meta", "meta_coord", "misc_attrib", "misc_feature", 
"misc_feature_misc_set", "misc_set", "object_xref", "oligo_array", 
"oligo_feature", "oligo_probe", "peptide_archive", "prediction_exon", 
"prediction_transcript", "protein_align_feature", "protein_feature", 
"qtl", "qtl_feature", "qtl_synonym", "repeat_consensus", "repeat_feature", 
"seq_region", "seq_region_attrib", "seq_region_mapping", "simple_feature", 
"splicing_event", "splicing_event_feature", "splicing_transcript_pair", 
"stable_id_event", "supporting_feature", "transcript", "transcript_attrib", 
"transcript_stable_id", "transcript_supporting_feature", "translation", 
"translation_attrib", "translation_stable_id", "unconventional_transcript_association", 
"unmapped_object", "unmapped_reason", "xref")), class = "data.frame", row.names = c(NA, 
-75L))
